### PR TITLE
Add login alert message data and api route

### PR DESCRIPTION
**Pull request by sebbalex**

### CODE REVIEW

**Comment by uolter** [comment]
@sebbalex we can also use terraform to publish the file in  s3:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_object

**Review by uolter — APPROVED** [review]

File: src/infra/api/oi.tpl.json [hunk 1]
```diff
@@ -223,6 +223,46 @@
...
+        ],
+        "description": "This route will serve the login alert messages",
+        "parameters": [],
+        "x-amazon-apigateway-integration": {
+          "credentials" : "${s3_apigateway_proxy_role}",
+          "httpMethod" : "GET",
+          "uri" : "${assets_bucket_uri}/login-alert-message.json",
+          "type" : "aws",          
+          "passthroughBehavior": "WHEN_NO_TEMPLATES",
+          "requestParameters": {
```
Comment (uolter) [inline src/infra/api/oi.tpl.json:239]: Is this useful? I mean: the json file has no logic capable to interpret any query string parameters  does it?

Comment (sebbalex) [reply src/infra/api/oi.tpl.json:239]: no, it's not. You're right, we don't definitely need those parameters. Thanks

**Comment by sebbalex** [comment]
@uolter I also added a correct response type and content, please check.

**Review by uolter — APPROVED** [review]

**Pull request merged**